src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -97,6 +103,9 @@ public FileResult getFileResult(String relativePath) {
         return FileInfoAnalyzer.analyzeFile(config, fileinfo);
     }
 
+    /**
+     * Asserts the correctness of file analysis.
```

Comment:
`Asserts the correctness of file analysis with regards to the contribution made by author named in {@code FAKE_AUTHOR_NAME}.`